lib/services/collections_service.dart

```diff
@@ -863,6 +881,16 @@ class CollectionsService {
       }
     }
   }
+
+  Future<List<Collection>> getSearchedCollections(String query) async {
```

Comment:
Instead of returning Collection, should we return `CollectionWithThumbnail` here? This way, you can simplify the rending of collections in the search result.

Reply:
Done